components/timeline/Timeline.tsx

```diff
@@ -12,7 +12,7 @@ export interface TimelineProps {
   pendingDot?: React.ReactNode;
   style?: React.CSSProperties;
   reverse?: boolean;
-  mode?: 'left' | 'alternate' | 'right';
+  mode?: 'left' | 'alternate' | 'right' | 'customize';
```

Comment:
No need to add `mode="customize"`, just follow postion of item in higher priority.

Reply:
That goot idea

Reply:
I think the mode shuld be in higher priority.